petabtests/cases/v1.0.0/sbml/0020/_conditions.tsv

```diff
@@ -0,0 +1,2 @@
+conditionId	A	B
+c0	initial_A	NaN
```

Comment:
It looks like it should be the string `"NaN"` here according to the docs, but actually I think it should be the empty string `""`, which becomes a `NaN` when read in by pandas.